src/content/development/_index.en.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "For Developers"
+title = "Development"
```

Comment:
Note that this change is going to break links in multiple places, especially outside of the website (like README and CONTRIBUTING files in the repos)

Reply:
Yep, I'll work on a big link cleaning after any major layout changes land.